startup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
do we want to have the standard `set ueo pipefail` ?

Reply:
I'll need to do a bit more work setting up the vars, but sure

Reply:
done